README.md

```diff
@@ -99,30 +99,33 @@ Contact: firstname.lastname@hpi.de
 
 ## Contributors
 
--	Yannick  Bäumer
--	Lawrence Benson
--	Timo     Djürken
--	Fabian   Dumke
--	Moritz   Eyssen
--	Martin   Fischer
--	Pedro    Flemming
--	Johannes Frohnhofen
--	Adrian   Holfter
--	Sven     Ihde
--	Michael  Janke
--	Max      Jendruk
--	Marvin   Keller
--	Sven     Lehmann
--	Jan      Mattfeld
--	Arne     Mayer
--	Torben   Meyer
--	Leander  Neiß
--	David    Schumann
--	Arthur   Silber
--	Daniel   Stolpe
--	Jonathan Striebel
--	Nils     Thamm
--	Carsten  Walther
--	Lukas    Wenzel
--	Fabian   Wiebe
--	Tim      Zimmermann
+-	Yannick   Bäumer
+-	Lawrence  Benson
+-	Timo      Djürken
+-	Fabian    Dumke
+-	Moritz    Eyssen
+-	Martin    Fischer
+-   Christian Flach
```

Comment:
Please replace the tabs in this entire file by spaces. It looks like no one ever manages to add this without breaking the formatting. Maybe its better if we use spaces (just like in the main code base).